src/libs/tmdk/src/commands/init.ts

```diff
@@ -337,52 +329,14 @@ async function import_scenes_and_models(
   return tmdk_config;
 }
 
-export const handler = async (argv: Arguments<Options>) => {
-  const workspaceId = argv["workspace-id"] as string;
-  const region = argv.region;
-  const out = argv.out;
-  const workspaceIdStr = workspaceId; //`${workspaceId}`; // TODO idiomatic
-  console.log(
-    `Bootstrapping project from workspace ${workspaceIdStr} in ${region} at project directory ${out}`
-  );
-
-  initDefaultAwsClients({ region: `${region}` }); // TODO idiomatic
-
-  // create directory if not exists
-  const outDir = `${out}`;
-  if (!fs.existsSync(outDir)) {
-    fs.mkdirSync(outDir); // TODO idiomatic
-  }
-
-  // create tmdk.json file
-  let tmdk_config: any = {
-    version: "0.0.2",
-  };
-  fs.writeFileSync(`${outDir}/tmdk.json`, JSON.stringify(tmdk_config, null, 4));
-
-  // TODO revisit: import workspace bucket/role (probably need role for specialized permissions)
-
-  // import component types
-  console.log("====== Component Types ======");
-  tmdk_config = await import_component_types(
-    workspaceIdStr,
-    tmdk_config,
-    outDir
-  );
-
-  // import scenes
-  console.log("====== Scenes / Models ======");
-  tmdk_config = await import_scenes_and_models(
-    workspaceIdStr,
-    tmdk_config,
-    outDir
-  );
-
-  // import entities
-  console.log("========== Entities =========");
+async function import_entities(
+  workspaceIdStr: string,
+  tmdk_config: tmdk_config_file,
+  outDir: string
+) {
   const entities = [];
   let nextToken: string | undefined = "";
-  let resp;
+  let resp: ListEntitiesCommandOutput;
```

Comment:
Nit: please rename variable to reflect that it is ListEntities response, such as `listEntitiesResp`

Reply:
remove the explicit type?

Reply:
I mean rename the variable, also same for `compResp` later in the code. If I am looking at this line later I may lose context as to what the response is from. We can rename both and save developer some pain, but not a priority to address